pages/en/billing.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Billing on Subgraph Studio
+---
+
+### Overview
+
+For a quick demo of how billing works on the Subgraph Studio, check out the video below:
+
+<VideoEmbed youtube="UrfIpm-Vlgs" />
+
+Invoices are statements of payment amounts owed by a customer and are typically generated on a weekly basis in the system. You'll be required to pay fees based on the query fees you generate using your API keys. While The Graph protocol operates on Ethereum Mainnet, the billing contract lives on the [Polygon](https://polygon.technology/) network to reduce transaction times and cost. It'll allow you to:
+
+- Add and remove GRT from your wallet
+- Keep track of your balances based on how much GRT you have added to your account, how much you have removed, and your invoices
+- automatically pay invoices based on query fees generated, as long as there is enough billing balance in your account.
+
+There are 4 states your invoice can be in:
+
+1. Pending - the invoice is in the queue to be paid from your billing balance
+2. Paid - your invoice has been successfully paid
+3. Unpaid - there is not enough GRT in your billing balance
+4. Partially Paid - only part of the invoice was paid, since there was not enough billing balance
+
+**See the diagram below for more information:**
+
+![Billing Flow](/img/billing-flow.png)
+
+### Adding GRT to Account
+
+Before making queries on The Graph Network, you will need to add GRT to your account. To add GRT to your account, take the following steps:
+
+1. Purchase GRT and ETH on an exchange of your choice
```

Comment:
Users can also acquire it via a decentralised exchange

Reply:
Got it!